### PR TITLE
Kapacitor release notes update - v1.6.3

### PR DESCRIPTION
**Pull request by lwandzura**
Closes #https://github.com/influxdata/docs-v2/issues/3700

Updates to Kapacitor release notes

### CODE REVIEW

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 1]
```diff
@@ -7,6 +7,20 @@ menu:
     name: Release notes
 ---
 
+## v1.6.3 [2022-01-20]
+
+### Features
+
+- Support for the `attributes` field in Alerta node.
+- Add BigPanda handler options.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:15]: Do we need to update the following docs to reflect these new configuration options/field available in event handlers?:

- [BigPanda event handler configuration](https://docs.influxdata.com/kapacitor/v1.6/event_handlers/bigpanda/) 
- [Alerta event handler](https://docs.influxdata.com/kapacitor/v1.6/event_handlers/alerta/)

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 2]
```diff
@@ -7,6 +7,20 @@ menu:
...
+- The `env var` config can now handle new items.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:16]: Does "handle" mean accept or add in this context? Do "items" refer to new env configurations?

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 3]
```diff
@@ -7,6 +7,20 @@ menu:
...
+- `Topic queue length` is now configurable.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:17]: Is the `Topic queue length` related to the `DefaultEventBufferSize = 5000` and `MinimumEventBufferSize = 1000`?

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 4]
```diff
@@ -7,6 +7,20 @@ menu:
...
+- Add new `address template` to email alert.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:18]: In the email [example TICKscript](https://docs.influxdata.com/kapacitor/v1.6/nodes/alert_node/#email), 
is it accurate to add the following as an example:

`.email('admin@example.com')`
` .toTemplates('oncall@example.com')`

And say that this passes the specified email addresses to the `email` property?

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md (hunk 1 above)
Comment (lwandzura) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:15]: @kelseiv I was thinking the same thing when working on this

Comment (kelseiv) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:15]: Maybe @docmerlin can help us clarify

Comment (timhallinflux) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:15]: I believe the answer is yes.

Comment (timhallinflux) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:15]: Update BigPanda text to better reflect the change -- >Add "host" and "attribute" options to BigPanda node and "auto-attributes" configuration option.

We should update the examples for using the Alerta handler and the BigPanda handler to demonstrate how to leverage these two new improvements.

Comment (timhallinflux) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:15]: review details for the BigPanda improvements here: https://github.com/influxdata/kapacitor/pull/2643

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md (hunk 3 above)
Comment (timhallinflux) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:17]: We need the config parameter names, default values and where it appears within the config file...

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md (hunk 4 above)
Comment (timhallinflux) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:18]: yeah, I'm really unclear about this new capability and how really to take advantage of it.  We need a better description and example within the docs.

**Review by timhallinflux — CHANGES_REQUESTED** [review]

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md (hunk 1 above)
Comment (lwandzura) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:15]: create issue https://github.com/influxdata/docs-v2/issues/3707 for BigPanda and Alerta event handler improvement/example updates

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md (hunk 2 above)
Comment (lwandzura) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:16]: Changed wording

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md (hunk 3 above)
Comment (lwandzura) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:17]: Will need to update the config page - https://github.com/influxdata/docs-v2/issues/3710

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md (hunk 4 above)
Comment (lwandzura) [reply content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:18]: Per Emrys - Previously, address had to be hardcoded - customer wanted to derive template for email address directly from data as opposed to script.

**Review by docmerlin — APPROVED** [review]

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 5]
```diff
@@ -7,6 +7,25 @@ menu:
     name: Release notes
 ---
 
+## v1.6.3 [2022-01-20]
+
+### Features
+
+- Add support for custom `attributes` field in [Alerta event handler](/kapacitor/v1.6/event_handlers/alerta/). 
+- Add `host` and `attribute` options to [BigPanda event handler](/kapacitor/v1.6/event_handlers/bigpanda/). 
+  - `host`: Corresponds to host alert payload parameter; identifies the main object that caused the alert.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:16]:
```suggestion
  - `host`: Identifies the main object that caused the alert.
```

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 6]
```diff
@@ -7,6 +7,25 @@ menu:
     name: Release notes
 ---
 
+## v1.6.3 [2022-01-20]
+
+### Features
+
+- Add support for custom `attributes` field in [Alerta event handler](/kapacitor/v1.6/event_handlers/alerta/). 
+- Add `host` and `attribute` options to [BigPanda event handler](/kapacitor/v1.6/event_handlers/bigpanda/).
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:15]:
```suggestion
- Add `host` and `attribute` options to [BigPanda event handler](/kapacitor/v1.6/event_handlers/bigpanda/):
```

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 7]
```diff
@@ -7,6 +7,25 @@ menu:
     name: Release notes
 ---
 
+## v1.6.3 [2022-01-20]
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:10]:
```suggestion
## v1.6.3 [2022-01-24]
```

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 8]
```diff
@@ -7,6 +7,25 @@ menu:
...
+
+### Features
+
+- Add support for custom `attributes` field in [Alerta event handler](/kapacitor/v1.6/event_handlers/alerta/). 
+- Add `host` and `attribute` options to [BigPanda event handler](/kapacitor/v1.6/event_handlers/bigpanda/). 
+  - `host`: Corresponds to host alert payload parameter; identifies the main object that caused the alert.
+  - `attribute`: Option to add additional attribute(s) to the alert payload.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:17]:
```suggestion
  - `attribute`: Adds additional attribute(s) to the alert payload.
```

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 9]
```diff
@@ -7,6 +7,25 @@ menu:
...
+- Add new `auto-attributes` configuration option to BigPanda node.
+-  Ability to add new headers to HTTP posts directly in `env var` config. 
+- `Topic queue length` is now configurable. This allows you to set a `topic-buffer-length` parameter in the Kapacitor config file in the
+[alert](https://docs.influxdata.com/kapacitor/v1.6/administration/configuration/#alert) section. The default is 5000.  Minimum length
+is 1000.
+- Add new `address template` to email alert. Email addresses no longer need to be hardcoded; can be derived directly from data.
+
+### Bug fixes
+- Deprecation in response to [sweet32](https://sweet32.info/): stopped using block ciphers.
+- Add additional detail to the error message `missing flux data`. This error is generated when issues occur when running a **Flux** query within a Batch TICKscript.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:27]:
```suggestion
- Add additional detail to the error message `missing flux data`. This error is generated when issues occur when running a **Flux** query within a batch TICKscript.
```

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 10]
```diff
@@ -7,6 +7,25 @@ menu:
...
+## v1.6.3 [2022-01-20]
+
+### Features
+
+- Add support for custom `attributes` field in [Alerta event handler](/kapacitor/v1.6/event_handlers/alerta/). 
+- Add `host` and `attribute` options to [BigPanda event handler](/kapacitor/v1.6/event_handlers/bigpanda/). 
+  - `host`: Corresponds to host alert payload parameter; identifies the main object that caused the alert.
+  - `attribute`: Option to add additional attribute(s) to the alert payload.
+- Add new `auto-attributes` configuration option to BigPanda node.
+-  Ability to add new headers to HTTP posts directly in `env var` config.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:19]:
```suggestion
- Ability to add new headers to HTTP posts directly in `env var` config. 
```

File: content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md [hunk 11]
```diff
@@ -7,6 +7,25 @@ menu:
...
+  - `attribute`: Option to add additional attribute(s) to the alert payload.
+- Add new `auto-attributes` configuration option to BigPanda node.
+-  Ability to add new headers to HTTP posts directly in `env var` config. 
+- `Topic queue length` is now configurable. This allows you to set a `topic-buffer-length` parameter in the Kapacitor config file in the
+[alert](https://docs.influxdata.com/kapacitor/v1.6/administration/configuration/#alert) section. The default is 5000.  Minimum length
+is 1000.
+- Add new `address template` to email alert. Email addresses no longer need to be hardcoded; can be derived directly from data.
+
+### Bug fixes
+- Deprecation in response to [sweet32](https://sweet32.info/): stopped using block ciphers.
```
Comment (kelseiv) [inline content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md:26]:
```suggestion
- Deprecated ciphers identified as "weak" in response to the [sweet32](https://sweet32.info/) attack.
```

**Review by kelseiv — APPROVED** [review]

**Pull request merged**